content/blog/2023/10/2023-10-20-twim.md

```diff
@@ -0,0 +1,318 @@
+>
+> ```json
+> {
+>   "contacts": [
+>     {
+>       "email_address": "your@email.here",
+>       "matrix_id": "@your:mxid.here",
+>       "role": "admin"
+>     }
+>   ]
+> }
+> ```
+>
+> If you're using [mdad](https://github.com/spantaleev/matrix-docker-ansible-deploy), there's a handy [guide](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-well-known.md#optional-introduction-to-homeserver-admin-contact-and-support-page) to assist you in setting this up.
+>
+> By taking this step, you'll be contributing to a safer and more secure Matrix Federation. We invite you to join us in the [#mrs:etke.cc](https://matrix.to/#/#mrs:etke.cc)  room, where you can connect with others who are interested in the MRS project.
+>
+> Let's work together to make the Matrix Federation a better place for all of us. Your support means a lot, and it's a step in the right direction for a more secure and connected future.
+
+<!-- more -->
+
+## Dept of Servers 🏢
+
+### Synapse ([website](https://github.com/matrix-org/synapse/))
+
+Synapse is a Matrix homeserver implementation developed by the matrix.org core team
+
+[clokep](https://matrix.to/#/@clokep:matrix.org) announces
+
+> This past week we released [Synapse 1.95rc1](https://github.com/matrix-org/synapse/releases/tag/v1.95.0rc1)! The overall theme seems to be
+> stability and performance. Although including no new features, it fixes a handful
+> of known issues, improves performance of purging rooms, and reduces the number of memory allocations of common tasks.
+>
+> Since the release candidate release we've been [preparing for Ubuntu 23.10](https://github.com/matrix-org/synapse/pull/16524) as well as landing additional bugfixes and [performance improvements](https://github.com/matrix-org/synapse/pull/16492).
+
+## Dept of Clients 📱
+
+### Element X iOS ([website](https://github.com/vector-im/element-x-ios))
+
+A total rewrite of Element-iOS using the Matrix Rust SDK underneath and targeting devices running iOS 16+.
+
+[Ștefan](https://matrix.to/#/@stefan.ceriu:matrix.org) announces
+
+> This week was all about continuing ongoing work:
+>
+> * We now have UI for [voice message recording](https://github.com/vector-im/element-x-ios/pull/1892)
+> * Audio and voice message file [waveforms in the timeline](https://github.com/vector-im/element-x-ios/pull/1919)
+> * Screens for [key backup and recovery](https://github.com/vector-im/element-x-ios/pull/1913)
+> * A very pretty [app lock screen](https://github.com/vector-im/element-x-ios/pull/1925)
+> * And last but not least a working, widget based, Element Call integration
+> ![An iOS dialog saying "Set up recovery" at the top, a description explaining the recovery key and a button to generate it. At the very bottom is a "Done" Button.](/blog/img/kxBZWSQHjOHIsFFJPFFzPhgy.png)
+> <video controls width="250"><source src="/blog/img/AVgKmzJozBGdUgaaDJcOOORd.mp4" type="video/mp4" /></video>
```

Comment:
```suggestion
>
> ![An iOS dialog saying "Set up recovery" at the top, a description explaining the recovery key and a button to generate it. At the very bottom is a "Done" Button.](/blog/img/kxBZWSQHjOHIsFFJPFFzPhgy.png)
>
> <video controls width="250"><source src="/blog/img/AVgKmzJozBGdUgaaDJcOOORd.mp4" type="video/mp4" /></video>
```

On my screen, it tries to throw them on the same line, but they're misaligned for some reason, so it looks weird.

Reply:
Yep good catch thanks!